.github/workflows/create-quantic-package.yml

```diff
@@ -46,20 +46,14 @@ jobs:
               repo: context.repo.repo,
               run_id: context.runId
             })
-      - name: Install NPM dependencies
-        working-directory: ./packages/quantic
-        run: |
-          npm i -g typescript
-          npm i -g sfdx-cli
-          npm i
+      - name: Setup
+        uses: ./.github/actions/setup
```

Comment:
One small warning about `setup`: it will install the version of node contained in `.nvmrc`, which is currently the version we tested for Atomic (v16.13.2)

Reply:
Ok thanks for the heads up. We haven't determined a version that is problematic for Quantic so I would go with this and correct if it causes an issue.